src/services/getFilters.js

```diff
@@ -0,0 +1,22 @@
+export default function getAllergiesLabels() {
```

Comment:
It seems you are returning not only allergies labels. Maybe consider renaming the function to something more fitting.